### ISSUE TITLE
Visual Studio 2015 - XAML Intellisense don't work

### ISSUE THREAD

**Issue by informaticaJugatoys** [UXDivers/Gorilla-Player-Support#124]
I just installed the latest version of "Gorilla Player" and the truth is that I was impressed.

The only problem is that editing an XAML file, don't work the intellisense.

I uninstalled "Gorilla Player" and again has returned to work.

I have installed VS 2015 Update 2, the latest version of Xamarin Forms (2.3) without any additional tool like Resharper.

I tested with two different computers and see the same problem. Both the sample project that provides Gorilla (MyCoolCompany), as the example of Xamarin (Phoneword)

Is there a problem with that? Can I suggest an alternative to intellisense?

I am initiating me with programming in Xamarin and these tools are helpful.

**Comment by LeoHere** [comment]
Hi. Please check that the solutions were successfully built at least once. XAML intellisense seems to relay on that. 

Gorilla does not interact with the XAML editor at all, so I don't see how we could being affecting. 

Hope it helps.

**Comment by msmrajesh** [comment]
Hi
Have you installed this. https://visualstudiogallery.msdn.microsoft.com/8195a8e2-a842-4389-a8cb-34e4672e2e13
Did you know by default Intellisense is not available for Xamarin unless you install this component.
For me it is working fine and Gorilla has no connection with the Intellisense. It is only to provide a preview of the xaml page on your device.
Thanks
Rajesh

**Comment by informaticaJugatoys** [comment]
I compiled back all the solution and the problem continues.

I tried it with a different computer recently installed (Windows 10 + VS 2015 Community Update 2) with the same result.

In case of not find a solution. Do you recommend me some kind of plugin that can replace the intellisense?

**Comment by informaticaJugatoys** [comment]
I installed the plugin indicated by msmrajesh and at first did not seem to work. But after recompiling the solution it seems that now works correctly.

As I stated in the first post, without having Gorilla Player installed, the autocomplete worked properly without the plugin, but installing Gorilla is when it stopped working. After uninstalling Gorilla intellisense work again it seems that the problem was related to this.

In any case, the plugin indicated by msmrajesh has solved the problem.

Thank you so much and I say again that I think Gorilla Player of great help especially for we are starting with Xamarin.
